### PR TITLE
 dx | 1636 | sre and test cases

### PR DESCRIPTION
**Pull request by harshithad0703**
- sanitized the required inputs 
- updated config.ts to run test cases (commented keepAlive from options as it is no longer supported)

### CODE REVIEW

File: src/stack.ts [hunk 1]
```diff
@@ -2017,6 +2017,9 @@ export class Stack {
 
   private async bindLeftoverAssets(queries: IQuery, locale: string, pointerList: IShelf[]) {
     // const contents = await readFile(getAssetsPath(locale) + '.json')
+    if (!this.sanitizeIQuery(queries)) {
+      throw new Error('Invalid queries provided');
+    }
```
Comment (aman19K) [inline src/stack.ts:2022]: @harshithad0703 , have you added this condition to fix test cases ?

Comment (harshithad0703) [reply src/stack.ts:2022]: @aman19K this condition is added for Snyk UI : Fail issue reported by SRE

Comment (aman19K) [reply src/stack.ts:2022]: @harshithad0703 , then it's better to convert this PR to draft and let @AniketDev7 first verify the changes. Pls explain him about fixes.
Hope you've already verified all changes. 

CC: @netrajpatel

File: src/stack.ts [hunk 2]
```diff
@@ -2376,5 +2385,30 @@ export class Stack {
...
+      return false;
+    }
+    if (!query || !Array.isArray(query.$or)) {
+      return false;
+    }
+    for (const item of query.$or) {
+      if (
+        typeof item._content_type_uid !== 'string' ||
+        typeof item.uid !== 'string' ||
+        (item._version && typeof item._version.$exists !== 'boolean') ||
```
Comment (aman19K) [inline src/stack.ts:2400]: @harshithad0703 , what if $exists doesn't exists?

Comment (harshithad0703) [reply src/stack.ts:2400]: will check other way for this

Comment (harshithad0703) [reply src/stack.ts:2400]:
```
interface IQuery {
  $or: Array < {
    _content_type_uid: string,
    _version?: {
      $exists: boolean,
    },
    uid: string,
    locale?: string,
  } >
}
```

this the type definition for IQuery

**Review by aman19K — APPROVED** [review]

**Pull request merged**